osquery/config/parsers/tests/options_tests.cpp

```diff
@@ -60,4 +60,20 @@ TEST_F(OptionsConfigParserPluginTests, test_unknown_option) {
   EXPECT_EQ(1U, doc["custom_fake"].GetUint());
   EXPECT_FALSE(Flag::getValue("custom_fake").empty());
 }
+
+TEST_F(OptionsConfigParserPluginTests, test_json_option) {
+  Config c;
+  std::map<std::string, std::string> update;
+
+  update["awesome"] =
+          "{\"options\": {\"complex\": {\"foo\": 1, \"bar\": \"baz\"}}}";
```

Comment:
You can use new [C++ string literal](https://en.cppreference.com/w/cpp/language/string_literal) here, if you want. It permit you to avoid escaping.
Smth like:
```C++
update["awesome"] = R"raw({
"options": {
  "complex": {
    "foo": 1, "bar": "baz"
  }
}
})raw";
```

Reply:
done, thanks for the hint